cardano-tx-generator/README.md

```diff
@@ -1 +0,0 @@
-# Transaction Generator
```

Comment:
So now we have not README.md at all ????
Why did you delete it. It did not do any harm.

Reply:
It wasn't pulling its meaning-per-byte weight, but sure let's get it back!